src/main/java/org/spongepowered/common/registry/CommonModuleRegistry.java

```diff
@@ -41,8 +41,51 @@
 import org.spongepowered.api.data.meta.PatternLayer;
 import org.spongepowered.api.data.persistence.DataFormat;
 import org.spongepowered.api.data.persistence.DataTranslator;
-import org.spongepowered.api.data.type.*;
```

Comment:
I went back through all the imports only to find that IntelliJ undid all of my changes... then I pressed the "undo" button and it immediately went back through and "fixed" the imports for me... :rage: